### PR TITLE
Docker setup

### PR DESCRIPTION
**Pull request by marcinlesek**
- Docker `dev` & `prod` setup,
- `prod` setup need to fill more details like `URL` for `.env` file and `nginx` conf,
- `prod` setup not tested,
- `README.md` updated

### CODE REVIEW

File: .docker/.env.dev.dist [hunk 1]
```diff
@@ -0,0 +1,9 @@
+COMPOSE_PROJECT_NAME=randomixer
```
Comment (whereiswolf) [inline .docker/.env.dev.dist]: Shouldn't `.env` files be in `.gitignore`?

File: .docker/.env.prod.dist [hunk 2]
```diff
@@ -0,0 +1,10 @@
+COMPOSE_PROJECT_NAME=randomixer
```
Comment (whereiswolf) [inline .docker/.env.prod.dist]: Shouldn't .env files be in .gitignore?

File: .docker/.env.dev.dist (hunk 1 above)
Comment (marcinlesek) [reply .docker/.env.dev.dist]: yes, `.env` is, here we have examples without any dangerous data.

File: .docker/.env.prod.dist (hunk 2 above)
Comment (marcinlesek) [reply .docker/.env.prod.dist]: response as above.

File: .docker/.env.dev.dist (hunk 1 above)
Comment (whereiswolf) [reply .docker/.env.dev.dist]: `.dist` suggests its for distribution, wouldn't `.example` be better?
Just asking xd

File: .docker/.env.prod.dist (hunk 2 above)
Comment (whereiswolf) [reply .docker/.env.prod.dist]: `.dist` suggests its for distribution, wouldn't `.example` be better?
Just asking xd

File: .docker/.env.dev.dist (hunk 1 above)
Comment (marcinlesek) [reply .docker/.env.dev.dist]: don't care about it, could be `.example` :smile:

**Pull request merged**